src/authentication/index.js

```diff
@@ -24,7 +25,7 @@ var DBConnection = require('./db-connection');
  * @param {String} [options.audience] identifier of the resource server who will consume the access token issued after Auth
  * @see {@link https://auth0.com/docs/api/authentication}
  */
-function Authentication(options) {
+function Authentication(auth0, options) {
```

Comment:
Is this an internal class?  Why did the `auth0` argument get added?  If its not internal, then that will cause breakages.  If its internal, can we document what's expected?

Reply:
it's internal. we need the webAuth instance to call the checkSession method.